examples/resources/random_string/import.sh

```diff
@@ -1,2 +1,13 @@
 # Strings can be imported by just specifying the value of the string:
-terraform import random_string.test test
\ No newline at end of file
+terraform import random_string.test test
+
+# Strings can be imported by just specifying the value of the string alongside the resource attributes.
+terraform import random_string.test "test length=4"
+
+# Random Password can be imported by specifying the value of the string alongside the resource attributes.
+terraform import random_string.test "test keepers=nil,length=4,special=true,upper=true,lower=true,number=true,min_numeric=0,min_upper=0,min_lower=0,min_special=0,override_special=_%@"
```

Comment:
Is this about random strings or passwords :thinking:

Reply:
Actually both, since the change has been made for string and passwords.

Reply:
It seems the context selection didn't go well. I meant to highlight the difference between the comment and the actual command.

Reply:
Copy paste issue, i'll fix this